.github/actions/setup_environment/action.yml

```diff
@@ -81,13 +81,11 @@ runs:
         insecure: true  # optional, use http instead of https. default false
         bucket: caching  # required
         use-fallback: false  # optional, use github actions cache fallback, default false
-        key: ${{ inputs.os }}-${{ inputs.cache_version }}-${{ inputs.python-version
-          }}-${{steps.date.outputs.week}}-${{inputs.install_integrations}}-${{ hashFiles('src/zenml/integrations/*/__init__.py')
-          }}
+        key: ${{ inputs.os }}-${{ inputs.python-version}}
         path: |
           ~/.cache/pip
         restore-keys: |
-          ${{ inputs.os }}-${{ inputs.cache_version }}-${{ inputs.python-version }}-${{steps.date.outputs.week}}-${{inputs.install_integrations}}-${{ hashFiles('src/zenml/integrations/*/__init__.py') }}
+          ${{ inputs.os }}-${{ inputs.python-version }}
```

Comment:
btw, maybe separate ticket / work, but this action has been deprecated a while back. https://github.com/syphar/restore-virtualenv. Maybe worth trying to get it all to work with `setup-python` which supposedly does most of the same things and is actually maintained?

see the note on the repo https://github.com/syphar/restore-virtualenv

Reply:
Let's do it somewhere else separately... we have a ticket in the backlog.